scripts/maintenance/requirements.txt

```diff
@@ -5,5 +5,6 @@ psycopg2-binary
 pydantic[email,dotenv]
 pylint
 python-dateutil
+tdqm
```

Comment:
Minor: I don't think you need tqdm since typer has an integrated progress bar. look for example in [https://github.com/ITISFoundation/osparc-simcore/blob/cbcc3c285c17d4ae2805a66c8a99e8436aadbbd7/scripts/maintenance/clean_projects_of_user.py](https://github.com/ITISFoundation/osparc-simcore/blob/cbcc3c285c17d4ae2805a66c8a99e8436aadbbd7/scripts/maintenance/clean_projects_of_user.py)
```python
with typer.progressbar(
                length=total, label="deleting projects"
            ) as progressbar:
```

Reply:
thanks for this, replaced with typer progressbar.